src/core/chart-container.tsx

```diff
@@ -28,7 +28,8 @@ interface ChartContainerProps {
   header?: React.ReactNode;
   filter?: React.ReactNode;
   navigator?: React.ReactNode;
-  legend?: React.ReactNode;
+  firstLegend?: React.ReactNode;
```

Comment:
I would rather use primaryLegend and secondaryLegend instead of first/second here instead.